Src/Base/AMReX_FBI.H

```diff
@@ -1,6 +1,12 @@
 #ifndef AMREX_FBI_H_
 #define AMREX_FBI_H_
 
+#include <AMReX_Array4.H>
+#include <AMReX_Box.H>
+#include <AMReX_Dim3.H>
+#include <AMReX_IntVect.H>
+
+
```

Comment:
This is meant to be a private header that is included by AMReX_FabArrayCommI.H`, which is also meant to be private.  But it's OK to include these.

Reply:
but... but...: https://github.com/include-what-you-use/include-what-you-use/blob/master/docs/WhyIWYU.md
:)